pkg/cmd/server/api/v1/conversions.go

```diff
@@ -43,6 +43,17 @@ func init() {
 				obj.PodEvictionTimeout = "5m"
 			}
 		},
+		func(obj *NodeConfig) {
+			// Ignore DeprecatedNetworkPluginName if NodeNetworkConfig.PluginName is provided
+			if len(obj.DeprecatedNetworkPluginName) != 0 && len(obj.NodeNetworkConfig.PluginName) == 0 {
+				obj.NodeNetworkConfig.PluginName = obj.DeprecatedNetworkPluginName
+			}
```

Comment:
is there ever a reason an MTU of 0 would be valid? if not, it's fine to default here